runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/streams/ServerIT.java

```diff
@@ -414,10 +412,13 @@ public void shouldUnbindRebind() throws Exception
         try
         {
             client5.connect(new InetSocketAddress("127.0.0.1", 12345));
-            fail("Client should not be able to connect, engine is at capacity");
+            ByteBuffer buf = ByteBuffer.allocate(1);
+            assertTrue(client5.read(buf) == -1);
+            client5.close();
         }
         catch (IOException ioe)
         {
+            ioe.printStackTrace();
```

Comment:
Now that we don't unbind, we should be able to remove the `try` `catch` block for this part , right?